compiler/test/formatter_outputs/ignores.gr

```diff
@@ -0,0 +1,20 @@
+let x = 5
+
+// formatter-ignore
+
+let x2 = [ 1, 
```

Comment:
This really caught me off guard before I realized there was a formatter-ignore haha

Reply:
:-D